comm/build.sbt

```diff
@@ -5,16 +5,28 @@ PB.targets in Compile := Seq(
   scalapb.gen(javaConversions = true) -> (sourceManaged in Compile).value
 )
 
+mainClass in (Compile, run) := Some("coop.rchain.comm.Main")
+mainClass in (Compile, packageBin) := Some("coop.rchain.comm.Main")
```

Comment:
Does this work for fat jar? I think there is should be definition of main class for "assembly". You can take a look at roscala's build.sbt

Reply:
It does work, but I've simplified it to exclude the above and only include assembly as you suggest.